Box.V2.Test/BoxEventsManagerTest.cs

```diff
@@ -88,6 +88,35 @@ public async Task GetUserEventsFile_ValidResponse()
             Assert.AreEqual(userFileEvents.Entries[0].Source.GetType(), typeof(BoxUserFileCollaborationEventSource));
             Assert.AreEqual(userFileEventSource.Id, "283257336425");
             Assert.AreEqual(userFileEventSource.Name, "ScreenShot2018-03-12at5.44.00PM.png");
+            Assert.AreEqual(userFileEventSource.UserEmail, "testuser@example.com");
+            Assert.AreEqual(userFileEventSource.OwnedBy.Id, "33333");
+            Assert.AreEqual(userFileEventSource.OwnedBy.Name, "Test User");
+        }
+
+        [TestMethod]
+        public async Task GetUserEventsFile_ExternalUser_ValidResponse()
+        {
+            var responseString = "{\"chunk_size\": 1, \"next_stream_position\": 123, \"entries\": [{\"source\":{\"file_id\":\"283257336425\",\"file_name\":\"ScreenShot2018-03-12at5.44.00PM.png\",\"user_email\":\"externaluser@box.com\",\"parent\":{\"type\":\"folder\",\"name\":\"AllFiles\",\"id\":\"0\"},\"owned_by\":{\"type\": \"user\",\"id\":\"33333\",\"name\": \"Test User\",\"login\":\"testuser@example.com\"}},\"created_by\":{\"type\":\"user\",\"id\":\"11111\",\"name\":\"Test User\",\"login\":\"test@user.com\"},\"created_at\":\"2018-03-16T15:12:52-07:00\",\"event_id\":\"85c57bf3-bc15-4d24-93bc-955c796217c8\",\"event_type\":\"COLLABORATION_INVITE\",\"ip_address\":\"UnknownIP\",\"type\":\"event\",\"session_id\":null,\"additional_details\":null}]}";
+            IBoxRequest boxRequest = null;
+            Handler.Setup(h => h.ExecuteAsync<BoxEventCollection<BoxEnterpriseEvent>>(It.IsAny<IBoxRequest>()))
+                .Returns(Task.FromResult<IBoxResponse<BoxEventCollection<BoxEnterpriseEvent>>>(new BoxResponse<BoxEventCollection<BoxEnterpriseEvent>>()
```

Comment:
Optional suggestion to improve readability for this would be to use the following instead of .`Returns(Task...)`:
```
    .ReturnsAsync(new BoxResponse<BoxEventCollection<BoxEnterpriseEvent>>()
                {
                    Status = ResponseStatus.Success,
                    ContentString = responseString
                }).Callback<IBoxRequest>(r => boxRequest = r);
```

Reply:
I think we should keep this so it will be consist with others test case.